### PR TITLE
feat: instrument telemetry for create command

### PR DESCRIPTION
**Pull request by Hweinstock**
## Description

Add telemetry instrumentation to the `create` command for both CLI and TUI paths.

- **CLI path:** wrapped `handleCreateCLI` with `runCliCommand('create', ...)` to emit CreateAttrs on success/failure
- **TUI path:** wrapped `useCreateFlow`'s `run()` with `withCommandRunTelemetry('create', ...)`
- Widened `CreateAttrs.agent_type` to use the shared `AgentType` enum (includes 'byo')
- Added telemetry assertions to existing integration tests

**Attributes emitted:** language, framework, model_provider, memory, protocol, build, agent_type, network_mode, has_agent

## Related Issue

Closes #

## Documentation PR

N/A

## Type of Change

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe):

## Testing

How have you tested the change?

- [x] I ran `npm run test:unit` and `npm run test:integ`
- [x] I ran `npm run typecheck`
- [x] I ran `npm run lint`
- [ ] If I modified `src/assets/`, I ran `npm run test:update-snapshots` and committed the updated snapshots

**Additional testing:**
- Integration tests pass (11/11 in create-frameworks + create-edge-cases)
- E2E verified with tarball: telemetry JSONL correctly emitted for both success and failure paths

## Checklist

- [x] I have read the CONTRIBUTING document
- [x] I have added any necessary tests that prove my fix is effective or my feature works
- [x] I have updated the documentation accordingly
- [x] I have added an appropriate example to the documentation to outline the feature, or no new docs are needed
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the
terms of your choice.

### CODE REVIEW

File: src/cli/tui/screens/create/useCreateFlow.ts [hunk 1]
```diff
@@ -181,7 +192,19 @@ export function useCreateFlow(cwd: string): CreateFlowState {
...
 
-    const run = async () => {
+    const attrs = {
+      language: standardize(Language, addAgentConfig?.language ?? 'Python'),
+      framework: standardize(Framework, addAgentConfig?.framework),
+      model_provider: standardize(ModelProvider, addAgentConfig?.modelProvider),
+      memory: standardize(MemoryEnum, addAgentConfig?.memory ?? 'none'),
+      protocol: standardize(Protocol, addAgentConfig?.protocol ?? 'HTTP'),
+      build: standardize(Build, addAgentConfig?.buildType ?? 'CodeZip'),
+      agent_type: (addAgentConfig?.agentType?.toLowerCase() ?? 'create') as 'create' | 'import',
```
[the remark this answers is not in the document]
Comment (Hweinstock) [reply src/cli/tui/screens/create/useCreateFlow.ts]: looks like a new feature since the schema was written, adding in now.

**Review by notgitika — APPROVED** [review]
looks great to me, thanks!
